packages/flutter/lib/src/widgets/app.dart

```diff
@@ -408,10 +412,21 @@ class WidgetsApp extends StatefulWidget {
   final Color color;
 
   /// {@template flutter.widgets.widgetsApp.locale}
-  /// The initial locale for this app's [Localizations] widget.
+  /// The initial locale for this app's [Localizations] widget is based
+  /// on this value.
+  ///
+  /// If the 'locale' is null then the system's locale value is used.
   ///
-  /// If the 'locale' is null the system's locale value is used.
+  /// The value of [Localizations.locale] will equal this locale if
+  /// it matches one of the [supportedLocales].
```

Comment:
This makes me wonder what it will be if it doesn't match... Maybe document that as well?

Reply:
Good point; updated.